letsencrypt/configuration.py

```diff
@@ -82,6 +84,21 @@ def http01_port(self):  # pylint: disable=missing-docstring
         else:
             return challenges.HTTP01Response.PORT
 
+    def determine_user_agent(self, authenticator, installer):
```

Comment:
Why don't we say, this is on my list of things to try to do today, but if it happens after merge/release it's a refactor ticket for the renewal milestone?